packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -1951,6 +1951,7 @@ export class StripeHelper extends StripeHelperBase {
           },
         ],
         proration_behavior: 'always_invoice',
+        proration_date: Math.floor(Date.now() / 1000),
```

Comment:
Unfortunately, this change alone won't resolve the issue.

As mentioned in [this Jira comment](https://mozilla-hub.atlassian.net/browse/FXA-8070?focusedCommentId=713450), on the ticket FXA-8070, to ensure that the proration amount shown at checkout, is the same when the subscription is upgraded, the same `proration_date` needs to be used.

Using `Date.now()` to calculate the `proration_date`, runs the risk of the user opening the checkout page, and only 30min or an hour later completing the subscription upgrade, which at that point `Date.now()` would have a different value that might cause a different proration amount, and hence a different value in the email.

Reply:
Thanks, Reino! Still having issues locally but I updated to get the current date at the start of the day so it will always match (provided the customer upgraded on the same day of their local time). Please let me know.

In edge cases such as a customer waiting half an hour and upgrading past midnight, it will fall into the next day and might again cause issues. Those scenarios fall out of scope for this ticket as we now have to look at how much time is provided to the customer on the payment page (i.e. if there is a time limit, and how we communicate that to the customer). If you have any suggestions or improvements I can make in this ticket, please let me know. Would also be down to pair if necessary.